### PR TITLE
Orthogonal initialization

### PR DESCRIPTION
**Pull request by martyn**
Attempt to increase the network depth capacity of both the Discriminator and Generator.

Inspired by https://arxiv.org/pdf/1702.08591.pdf

### CODE REVIEW

File: examples/2d-measure-accuracy.py [hunk 1]
```diff
@@ -33,17 +33,23 @@ def custom_generator_config():
...
     }
 
+
 def custom_discriminator(gan, config, x, g, xs, gs, prefix='d_'):
     net = tf.concat(axis=0, values=[x,g])
-    net = linear(net, 128, scope=prefix+'lin1')
-    net = tf.nn.relu(net)
-    net = linear(net, 128, scope=prefix+'lin2')
+    net = linear(net, 128, scope=prefix+'linone')
+    for i in range(0):
```
Comment (martyn) [inline examples/2d-measure-accuracy.py]: Remove this.

File: examples/2d-measure-accuracy.py [hunk 2]
```diff
@@ -362,14 +410,14 @@ def modes(x):
...
 
-            if(i > 490000):
+            if(i > 9000):
                 ax, ag, agg, dl = gan.sess.run([accuracy_x_to_g, accuracy_g_to_x, accuracy_g_to_g, gan.graph.d_log], {gan.graph.x: x_0, gan.graph.z[0]: z_0})
                 diversity += agg
                 ax_sum += ax
                 ag_sum += ag
                 dlog = dl
 
-        with open("results.csv", "a") as myfile:
```
Comment (martyn) [inline examples/2d-measure-accuracy.py]: This example script is pretty hacky.  The tech debt impact isn't huge, but it's a really cool script and we should make it a bit easier to use.  I'll add a note to https://www.pivotaltracker.com/story/show/141733753

File: examples/2d-test.py [hunk 3]
```diff
@@ -57,15 +57,20 @@ def no_regularizer(amt):
  
 def custom_discriminator(gan, config, x, g, xs, gs, prefix='d_'):
     net = tf.concat(axis=0, values=[x,g])
-    net = linear(net, 128, scope=prefix+'lin1')
-    net = tf.nn.relu(net)
-    net = linear(net, 128, scope=prefix+'lin2')
+    net = linear(net, 128, scope=prefix+'linone')
+    for i in range(24):
```
Comment (martyn) [inline examples/2d-test.py]: We probably don't want this here.

File: hypergan/util/ops.py [hunk 4]
```diff
@@ -261,13 +263,16 @@ def lrelu_sq(x):
     dim = len(x.get_shape()) - 1
     return tf.concat(axis=dim, values=[lrelu(x), tf.minimum(tf.abs(x), tf.square(x))])
 
-def linear(input_, output_size, scope=None, mean=0., stddev=0.02, bias_start=0.0, with_w=False, regularizer=None):
+def linear(input_, output_size, scope=None, mean=0., stddev=0.02, bias_start=0.0, with_w=False, regularizer=None, gain=1.0):
     shape = input_.get_shape().as_list()
 
+    initializer = tf.orthogonal_initializer(1.0)
```
Comment (martyn) [inline hypergan/util/ops.py]: Initialization seems like it might be a hyperparam.

This might be related to https://www.pivotaltracker.com/story/show/142115353
